packages/ui-extensions/src/surfaces/admin/components/patterns/examples/indexTable.html

```diff
@@ -0,0 +1,142 @@
+<s-section padding="none" accessibilityLabel="Puzzles table section">
+  <s-table fullwidth>
+    <s-grid slot="filters" gap="small-200" gridTemplateColumns="1fr auto">
+      <s-text-field
+        icon="search"
+        placeholder="Searching all puzzles"
+      ></s-text-field>
+      <s-button
+        icon="sort"
+        variant="secondary"
+        interestFor="sort-tooltip"
+        commandFor="sort-actions"
+      ></s-button>
+      <s-tooltip id="sort-tooltip">
+        <s-text>Sort</s-text>
+      </s-tooltip>
+      <s-popover id="sort-actions">
+        <s-stack gap="none">
+          <s-box padding="small">
+            <s-choice-list label="Sort by" name="Sort by" details="">
+              <s-choice
+                label="Product title"
+                value="product-title"
+                selected
+              ></s-choice>
+              <s-choice label="Created" value="created"></s-choice>
+              <s-choice label="Updated" value="updated"></s-choice>
+              <s-choice label="Inventory" value="inventory"></s-choice>
+              <s-choice label="Product type" value="product-type"></s-choice>
+              <s-choice
+                label="Publishing error"
+                value="publishing-error"
+              ></s-choice>
+              <s-choice label="Vendor" value="vendor"></s-choice>
+            </s-choice-list>
+          </s-box>
+          <s-divider></s-divider>
+          <s-box padding="small">
+            <s-choice-list label="Order by" name="Order by" details="">
+              <s-choice label="A-Z" value="product-title" selected></s-choice>
+              <s-choice label="Z-A" value="created"></s-choice>
+            </s-choice-list>
+          </s-box>
+        </s-stack>
+      </s-popover>
+    </s-grid>
+    <s-table-header-row>
+      <s-table-header listSlot="primary">
+        <s-stack direction="inline" gap="small-200" align="center">
+          <s-checkbox accessibilityLabel="Select all puzzles"></s-checkbox>
```

Comment:
Unsure: I'd think that we'd want the checkboxes to be in their own separate `s-table-header` / `s-table-cell` instead of being attached to the first element in the row.